mappings/net/minecraft/util/registry/BuiltinRegistries.mapping

```diff
@@ -56,3 +64,7 @@ CLASS net/minecraft/class_5458 net/minecraft/util/registry/BuiltinRegistries
 		ARG 0 registry
 		ARG 1 id
 		ARG 2 value
+	CLASS class_7488 Initializer
+		COMMENT A functional interface that initializes the registry and returns the default value.
+		METHOD run (Lnet/minecraft/class_2378;)Lnet/minecraft/class_6880;
```

Comment:
```suggestion
		METHOD initialize (Lnet/minecraft/class_2378;)Lnet/minecraft/class_6880;
```

Reply:
This is deobfscated name, I don’t think we can rename.

Reply:
I think we can, and the enigma form would be:
```
METHOD run initialize (Lnet/minecraft/class_2378;)Lnet/minecraft/class_6880;
```
but I don't know if our toolchain handles functional interfaces' method renames properly.